Sources/Resources/Assets.xcassets/Expandable/icon-expandable-chevron.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "icon-expandable-chevron.svg",
+      "idiom" : "universal"
```

Comment:
Should this be enabled?
<img width="262" alt="Skærmbillede 2024-07-02 kl  11 28 50" src="https://github.com/warp-ds/warp-ios/assets/654037/240a5f5e-58dc-41f8-8630-307d5c578605">

Reply:
Good question. It is not at the moment and doesn't seem to be in any of the icons in the Assets folder.

From what I read here:

https://forums.developer.apple.com/forums/thread/694526

> When i tick the 'Preserve Vector Data' box, the asset is scaled at runtime using the vector data, allowing for smooth scaling and crisp images at any scale.

So that kinda indicate that it should be checked...but then we should probably do that on all images.

Have you had any discussions previously related to this @mohsen-schibsted?

Reply:
No we have not started working on any icons. Our designers are working on our icon v2.0 and when that is ready we will start.

Reply:
Right, I'll wait till the icons have settled then 👍